### ISSUE TITLE
Maven dependency net.sourceforge.kxml not found

### ISSUE THREAD

**Issue by GoogleCodeExporter** [josaluntu/ksoap2-android#13]
```
A repo must be manually added. I had to add:

<repository>
            <id>ibiblio</id>
            <url>http://mirrors.ibiblio.org/pub/mirrors/maven2</url>
        </repository>
```

Original issue reported on code.google.com by `Karsten....@googlemail.com` on 27 Jan 2010 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
The same applies to kobjects and me4se.

The URL is wrong. I had to upload it into my private repo.

```

Original comment by `Karsten....@googlemail.com` on 27 Jan 2010 at 11:52

**Comment by GoogleCodeExporter** [comment]
```
I have placed the missing artifacts here:

http://globalplatform.sourceforge.net/repo/m2-repository/net/sourceforge/

```

Original comment by `Karsten....@googlemail.com` on 28 Jan 2010 at 1:10

**Comment by GoogleCodeExporter** [comment]
```
This issue was resolved with the 2.4 release.
```

Original comment by `karl.mic...@gmail.com` on 7 Mar 2010 at 3:37
- Changed state: **Fixed**
